README.md

```diff
@@ -41,6 +41,23 @@ The following vendors export data in SNIRF format.
 | Cortivision    | https://www.cortivision.com      |
 
 
+## Governance
+
+SNIRF is a community guided project with oversight from academic and industry partners.
+The governance of the project is described in the [government.md](GOVERNMENT.md) document.
+The current steering committee is:
+
+| Name           |          Institution             |
+|----------------|----------------------------------|
+| David Boas     | Boston University                |
+| Robert Luke    | AE Studio                        |
+| Samuel Powell  | Gowerlabs                        |
+| Jörn Horschig  | Artinis Inc                      |
```

Comment:
```suggestion
| Jörn Horschig  | Artinis Medical Systems B.V. |
```

Reply:
Sorry about that! I copied that text from the specification document, so we should update your affiliation there too